stream-chat-android-ui-components/src/main/res/values/attrs_giphy_media_attachment_view.xml

```diff
@@ -25,6 +25,32 @@
         <attr name="streamUiGiphyMediaAttachmentPlaceHolderIcon" format="reference" />
         <!-- Media attachment background color -->
         <attr name="streamUiGiphyMediaAttachmentImageBackgroundColor" format="color|reference" />
+        <!-- Determines the sizing strategy of Giphy containers -->
+        <attr name="streamUiGiphyMediaAttachmentSizingMode" format="enum">
+            <!-- Automatically sizes the container according to the available
+             width and height while respecting the original Giphy aspect ratio.
+
+             Choosing this value makes the container ignore attributes
+             'streamUiGiphyMediaAttachmentWidth', 'streamUiGiphyMediaAttachmentHeight' and
+             'streamUiDimensionRatio'. -->
+            <enum name="automatic_resizing" value="0" />
```

Comment:
Let me know if you think `adaptive_resizing` is better.
It came to me only after raising the PR.

Reply:
I'm going to rename this in the morning to just `adaptive`.
I think it's a better name, minimizes repetition and sounds nicer.

Reply:
yeah adaptive is better, so feel free to rename to amtch compose